containers/EmpSponsors.ts

```diff
@@ -121,7 +122,7 @@ const useEmpSponsors = () => {
           let newLiquidations: LiquidationMap = {};
 
           const collReqFromWei = parseFloat(
-            fromWei(collateralRequirement, collDecs)
+            fromWei(collateralRequirement, tokenDecs)
```

Comment:
This section is still a WIP because the subgraph currently does not account for non 18 decimal precision